connectors/v2/youtube.js

```diff
@@ -14,10 +14,28 @@ chrome.storage.local.get('Connectors', function(data) {
 	}
 });
 
-Connector.playerSelector = '#page';
+Connector.videoSelector = '#player-api .html5-main-video';
+
+{
+	$(Connector.videoSelector).on('timeupdate', () => {
+		if (Connector.isStateChangeAllowed()) {
```

Comment:
To show that's a separate block for setup(not sure if it's needed because the whole file is connector setup)